docs/release-notes/platform_10-6-0_releasenotes.md

```diff
@@ -0,0 +1,158 @@
+---
+id: v10.6.0
+title: Version 10.6.0 Release Notes
+sidebar_label: v10.6.0
+image: "img/release-note-link-preview.png"
+---
+
+_March 16, 2023_
+
+## Console
+
+### New Features
+
+In this Console release we have brought exciting New Features which are currently available exclusively for PaaS users. 
+Here below you can discover all of them!
+
+#### Fast Data No Code Configuration for ER Schema (only available for PaaS users)
+
+:::info This feature is still in BETA, it is under active development. :::
+
+Are you tired of writing ER Schemas manually? The new Fast Data ER Schema No Code Configuration is now generally available!
+ 
+It is now possible to create and generate relationships between Projections using a simple and easy to use No Code configuration.  
+This tool is able to generate a JSON that will be used by the Single View Creator. The generated JSON is exactly the same JSON that would have been written manually in the Low Code Configuration.
+
+In order to exploit this functionality you have to open the Projections section and navigate in the new ER Schema tab. You can create a new ER Schema and start designing it!   
+When configuring the Single View Creator, in the Single Views section, you just have to select one of the ER Schemas created in the aforementioned section from a dropdown list.
+
+This feature will only be available for PaaS users and it can be activated only for specific projects. If you want to try it out, contact your system administrator!
+Do not miss out the [official documentation](/fast_data/configuration/config_maps/erSchema.md#use_the_no_code)! 
+
+![Fast Data No Code](./img/10.6/fast_data_no_code.png)
+
+
+#### Feature Preview section (only available for SaaS Console users)
+
+SaaS users can now have visibility of features in Console that are still in preview and decide to try them out and give us precious feedback.  
+For instance, once entered in this section, PaaS users can now try the new `Dynamic Sidebar` of the Design Area.   
+To discover more, just try this new feature and tell us what you think!
+ 
+This section can be accessed from the Launchbar by clicking on the `Feature Preview` item inside the dropdown menu that appears when hovering over the user avatar. 
+
+![Feature Preview Modal - Feature Preview modal with some features enabled](./img/10.6/feature-preview.png)
+
+#### User Preferences section (only available for PaaS users)
```

Comment:
This section will only be published if we get the updated privacy policy for email notifications; if we don't get the privacy policy we won't release neither the email notification feature nor this feature (which currently is only meaningful if the email notification feature is active)